code/obj/item/card.dm

```diff
@@ -82,6 +82,7 @@ GAUNTLET CARDS
 	var/title = null
 	var/emagged = 0
 	var/datum/reagent_group_account/reagent_account = null
+	var/keep_icon = 0
```

Comment:
document what this variable does, and use `TRUE`/`FALSE`

Reply:
sure thing zewaka